src/lib.rs

```diff
@@ -106,7 +106,8 @@ where
     }
 
     fn flush(&mut self) -> Result<(), E> {
-        for _ in 0..20 {
+        // Should be > 300μs, so for an SPI Freq. of 3.8MHz, we have to send at least 1140 low bits or 140 low bytes
+        for _ in 0..140 {
```

Comment:
I actually had to tweak this to 160 this morning.  I wonder if we could introduce a `reset_offset` :thinking:

Reply:
Yeah. I'll follow up with an issue if anything is weird when I get it on the analyzer. I just wanted to leave a note for now just for posterity. Most of the animations I did were working as expected, and it was just the main one I had running with rtic that wasn't working, so there's plenty of room for side-effects to be introducing problems.

https://github.com/LongLiveCHIEF/led-backlighting-controller/tree/4a60db2e930364bc456b67322aa68219d13b29d4  (both the examples work as expected, but the main had the first led always green until I changed to `160`)

Reply:
so... looks like my particular led's reset is right at 375μs.  Some of my code is using rtic, and that code is resulting in a reset time of just under 375μs, and some of my code (non-rtic) is resulting in a reset time of 382μs.

...which basically just means 141 instead of 140 works.  I'm digging more into the clock timing in my rtic code, since I think this adjustment can be made in my code.

Reply:
... I just slowed down the spi clock to ~2600 khz.